examples/docs/en-US/menu.md

```diff
@@ -62,7 +62,7 @@ Top bar NavMenu can be used in a variety of scenarios.
 
 Vertical NavMenu with sub-menus.
 
-::: demo
+::: demo you can use `el-menu-item-group` component to create the menu group，the name of the group is setting by the `title` prop or the name slot.
```

Comment:
You can use the `el-menu-item-group` component to create a menu group, and the name of the group is determined by the `title` prop or a named slot.